repository/src/main/resources/db/migration/V20210818_12761__RDM-12761_Update_base_type_field_ids_for_WorkAllocation.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE complex_field SET reference = 'region' WHERE reference='RegionId' and complex_field_type_id = (select id from field_type where reference = 'CaseLocation' and jurisdiction_id is null);
+
+UPDATE complex_field SET reference = 'baseLocation' WHERE reference='BaseLocationId' and complex_field_type_id = (select id from field_type where reference = 'CaseLocation' and jurisdiction_id is null);
```

Comment:
should we be updating the label as well to remove the ID reference? https://github.com/hmcts/ccd-definition-store-api/blob/master/repository/src/main/resources/db/migration/V20201218__RDM-10041__CaseLocation.sql

Reply:
updated the label as well